apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Footer/Reactions/Reactions.tsx

```diff
@@ -0,0 +1,89 @@
+import Stack from '@mui/material/Stack'
+import { useTranslation } from 'next-i18next'
+import { ReactElement } from 'react'
+
+import { useCommand } from '@core/journeys/ui/CommandProvider'
+import { useJourney } from '@core/journeys/ui/JourneyProvider'
+import { GetJourney_journey } from '@core/journeys/ui/useJourneyQuery/__generated__/GetJourney'
+import Share from '@core/shared/ui/icons/Share'
+
+import { useJourneyUpdateMutation } from '../../../../../../../libs/useJourneyUpdateMutation'
+import { Accordion } from '../../Properties/Accordion'
+
+import { ReactionOption } from './ReactionOption'
+
+export type ReactionFields = keyof Pick<
+  GetJourney_journey,
+  'showShareButton' | 'showLikeButton' | 'showDislikeButton'
+>
+export type UpdateReactionInput = { [key in ReactionFields]?: boolean }
+
+export function Reactions(): ReactElement {
+  const { t } = useTranslation('apps-journeys-admin')
+  const { journey } = useJourney()
+  const [journeyUpdate] = useJourneyUpdateMutation()
+  const { add } = useCommand()
+
+  function handleToggle(input: UpdateReactionInput): void {
+    if (journey == null) return
+
+    const undoInput = Object.fromEntries(
+      Object.entries(input).map(([key, value]) => [key, !value])
+    )
+
+    add({
+      parameters: {
+        execute: { input },
+        undo: { input: undoInput }
+      },
+      execute({ input }) {
+        void journeyUpdate({
+          variables: {
+            id: journey.id,
+            input
+          },
+          optimisticResponse: {
+            journeyUpdate: {
+              __typename: 'Journey',
+              id: journey.id,
+              title: journey.title,
+              description: journey.description,
+              strategySlug: journey.strategySlug,
+              language: journey.language,
+              tags: [],
```

Comment:
For https://github.com/JesusFilm/core/pull/3320/files#r1733573451
```suggestion
              ...journey,
```